src/background/services/monetization.ts

```diff
@@ -256,13 +274,28 @@ export class MonetizationService {
     await this.resumePaymentSessionsByTabId(currentTab.id);
   }
 
-  async toggleWM() {
-    const { continuousPaymentsEnabled } = await this.storage.get([
+  async toggleContinuousPayments() {
+    const { continuousPaymentsEnabled, enabled } = await this.storage.get([
       'continuousPaymentsEnabled',
+      'enabled',
     ]);
     const nowEnabled = !continuousPaymentsEnabled;
     await this.storage.set({ continuousPaymentsEnabled: nowEnabled });
-    if (nowEnabled) {
+    if (nowEnabled && enabled) {
```

Comment:
qq: why do we have to check both `continuousPaymentsEnabled` and `enabled` here? 🤔 
`enabled` is the bool for the entire extension monetization, right?

Reply:
If user has disabled continuous payments from toggle in Settings > Rate, but enabled is true, payments are still not to get through.